Library/Homebrew/cask/cmd/abstract_command.rb

```diff
@@ -12,7 +12,7 @@ class AbstractCommand
       option "--verbose",         :verbose,       false
       option "--outdated",        :outdated_only, false
       option "--require-sha",     :require_sha,   false
-      option "--[no-]quarantine", :quarantine,    true
+      option "--[no-]quarantine", :quarantine,    false
 
```

Comment:
Revert all the opt-in changes. (I understand you were planning to do that already.)